### ISSUE TITLE
Spatial data in cell browser

### ISSUE THREAD

**Issue by cancerGen** [maximilianh/cellBrowser#259]
Hi 
I was wondering if there is any way to use/explore spatial data in cell browser? Something similar to Loupe browser if possible

Thanks

**Comment by maximilianh** [comment]
Yes I’m just committing it! I have 15 spatial datasets up, but hidden
because they’re not my datasets.

Is your file 10X visium or something else ? Could you share it so I can
test it ? Can I follow up by direct email in case it’s not published yet ?

**Comment by cancerGen** [comment]
Hi,
I am actually using public data and have not generated any data myself. Can I check out the browser?

**Comment by maximilianh** [comment]
Can you point me to some spatial public dataset such that I can use it to
show you the feature? The only two datasets I have now are pre-publication
and their authors do not want me to share the links. But the importers
handle 10X just fine, so hopefully adding a spatial dataset won't be too
much work. Thanks for your patience with this, the holidays delayed the
response.

**Comment by cancerGen** [comment]
Hi, 
So I am using this data from the SPATA paper. The data download tutorial is provided here: https://github.com/theMILOlab/SPATAData

If you could load any one of these samples for me to explore the browser it will be very helpful. Thanks

**Comment by maximilianh** [comment]
Uhhh.... I see. But I've never seen this data format... Do you know if the
SPATA project has normal 10X Visium .h5 files somewhere? Is this actually
Visium data?

**Comment by cancerGen** [comment]
It is indeed 10x data. This is the original publication I think for your reference: https://pubmed.ncbi.nlm.nih.gov/35700707/.
You can download the SPATA2 objects using R and convert it into Seurat object using the following function.
https://rdrr.io/github/kueckelj/SPATA2/man/transformSpataToSeurat.html

Could you please tell me what is supposed to be the input format for the UCSC spatial data browser? I could write a small script to help you get the files in a correct format.

**Comment by maximilianh** [comment]
I dropped the ball on this sorry. The spatial part right now can read
either h5AD scanpy images or just normal jpeg or PNG files with the min/max
range specified of the overlaid coordinates. It will scale the coordinates.

So just providing the image in the cellbrowser.conf file should work in
most cases. We still have not released the code, but you can always use the
"develop" branch..

**Comment by maximilianh** [comment]
Alternatively, you can just send us a Seurat or Scanpy object and the
spatial image behind it in any image format.

**Comment by maximilianh** [comment]
Oh by the way that code is committed now and the example cellbrowser configuration should have an example.

What i don't have yet is a sample dataset... maybe I should use a pbmc3k-spatial or some similar common demo dataset...
